lib/resty/kafka/errors.lua

```diff
@@ -1,24 +1,115 @@
 -- Copyright (C) Dejiang Zhu(doujiang24)
 
-
+-- Reference: https://kafka.apache.org/protocol.html#protocol_error_codes
 local _M = {
-    [0]     = 'NoError',
-    [-1]    = 'Unknown',
-    [1]     = 'OffsetOutOfRange',
-    [2]     = 'InvalidMessage',
-    [3]     = 'UnknownTopicOrPartition',
-    [4]     = 'InvalidMessageSize',
-    [5]     = 'LeaderNotAvailable',
-    [6]     = 'NotLeaderForPartition',
-    [7]     = 'RequestTimedOut',
-    [8]     = 'BrokerNotAvailable',
-    [9]     = 'ReplicaNotAvailable',
-    [10]    = 'MessageSizeTooLarge',
-    [11]    = 'StaleControllerEpochCode',
-    [12]    = 'OffsetMetadataTooLargeCode',
-    [14]    = 'OffsetsLoadInProgressCode',
-    [15]    = 'ConsumerCoordinatorNotAvailableCode',
-    [16]    = 'NotCoordinatorForConsumerCode',
+    [0]     = {error = 'NONE', retriable = false},
```

Comment:
`error` is a global function in Lua land. I think we'd better change it to another name.

Reply:
how about `err_msg` or `msg`?

Reply:
`msg` better